typing/mtype.ml

```diff
@@ -134,28 +134,28 @@ let strengthen_decl ~aliasable env md p =
              (Subst.Lazy.of_module_decl md) p in
   Subst.Lazy.force_module_decl md
 
-let rec make_aliases_absent pres mty =
+let rec make_aliases_present pres mty =
   match mty with
-  | Mty_alias _ -> Mp_absent, mty
+  | Mty_alias _ -> Mp_present, mty
   | Mty_signature sg ->
-      pres, Mty_signature(make_aliases_absent_sig sg)
+      pres, Mty_signature(make_aliases_present_sig sg)
   | Mty_functor(arg, res) ->
-      let _, res = make_aliases_absent Mp_present res in
+      let _, res = make_aliases_present Mp_present res in
       pres, Mty_functor(arg, res)
   | mty ->
       pres, mty
 
-and make_aliases_absent_sig sg =
+and make_aliases_present_sig sg =
   match sg with
     [] -> []
   | Sig_module(id, pres, md, rs, priv) :: rem ->
-      let pres, md_type = make_aliases_absent pres md.md_type in
+      let pres, md_type = make_aliases_present pres md.md_type in
```

Comment:
These functions can now have a simpler implementation. They no longer care what the `md_type` is, they are just always going to set the presence to `Mp_present`. So `make_aliases_present` doesn't need to take a `pres` parameter or return a presence anymore.